src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -857,16 +862,15 @@ export default {
       const header = filing?.header
       const correction = filing?.correction
 
-      const filingDateTime = this.apiToSimpleDateTime(header.date)
-      const filingDate = filingDateTime?.slice(0, 10)
+      const filingDateTime = this.apiToPacificDateTime(header.date)
+      const filingDate = this.apiToPacificDate(header.date)
 
       if (header && correction) {
         const item: HistoryItemIF = {
           filingType: FilingTypes.CORRECTION,
           title: `Correction - ${this.filingTypeToName(correction.correctedFilingType)}`,
           filingId: header.filingId,
           filingAuthor: header.certifiedBy || 'Registry Staff',
-          filingDateTime: this.apiToSimpleDateTime(header.date), // used for receipt
```

Comment:
This property is not used in this history object. It is used in the receipt object below (line 888).